examples/Example_domain/datapackage.json

```diff
@@ -1,72 +1,85 @@
 {
-  "name": "Example_domain",
-  "title": "Example reef cluster data package",
-  "description": "Example data package for ADRIA",
-  "version": "0.4.0-rc",
-  "sources": [],
-  "simulation_metadata": {
-    "timeframe": [2025, 2099]
-  },
-  "contributors": [
+"name": "Example_domain",
```

Comment:
The standard for JSON is to indent using 2 spaces, not four.

Reply:
Also, shouldn't this be updated to `Test_example` if they are intended for testing?

Reply:
This is still formatted incorrectly.